docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,55 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+Failure modes encountered when abstracting
+--------------------------------------------------
+
+In "Fairness and Abstraction in Sociotechnical Systems" Selbst et al. [#4]_ identify failure modes that arise the applying abstraction. They identify them as:
```

Comment:
Did you mean "that arise from applying abstraction"?

Reply:
Yes! Solved